src/content/en/2021/jamstack.md

```diff
@@ -441,8 +487,13 @@ Lodash is heavily used and was present among all SSGs up to 10% for Gatsby.
   sql_file="resource_weights.sql"
 ) }}
 
+{# TODO - this figure description should be described more #}
+
 On the other hand, CSS is slightly heavier than Hugo and Jekyll.
-Since of the benefits of styled-components is clean none duplicated CSS, it can be a good explaning why CSS size for these JavaScript SSGs are lower. One more hypothesis, old fashined SSGs use old fashion way for handling interactions and animations using CSS. JavaScript based, use more JavaScript in ganaral, hence might use it to relpace functionlity can be implemented with CSS.
+Since of the benefits of styled-components is clean none duplicated CSS, it can be a good explaining why CSS size for these JavaScript SSGs are lower. One more hypothesis, old fashined SSGs use old fashion way for handling interactions and animations using CSS. JavaScript based use more JavaScript in ganaral, hence it might be used to replace functionlity that can be implemented with CSS.
+
+{# TODO - "none duplicated"? not "non"? #}
```

Comment:
```suggestion
Since of the benefits of styled-components is clean, non-repetitive CSS, this could explain why CSS size for these JavaScript SSGs are lower. One more hypothesis is that old fashioned SSGs use old fashion methods for handling interactions and animations using CSS. JavaScript-based SSGs use more JavaScript in general, hence they might more often be used to replace functionality that could be implemented with CSS.
```